backend/app/main/survey_views.py

```diff
@@ -22,24 +19,12 @@ def sms_signup():
     
     if 'question_id' in session:
         # Retrieve the Client's phone number and format it
-        phone_number = request.values.get('From', None)
-        phone = "%2B" + phone_number[1: ]
-
-        # Check if record already exists
-        prev_response = requests.get( 
-            "https://api.airtable.com/v0/appw4RRMDig1g2PFI/SMS%20Responses?filterByFormula={Phone_Number}='"+phone+"'",
-            headers={"Authorization": str(os.environ.get("API_KEY"))})
-        response_id = "NONE"
-
-        if prev_response.status_code == 200:
-            response_json = prev_response.json()
-            for r in response_json["records"]:
-                response_id = r["id"]
-
+        phone_number = "%2B" + request.values.get('From', None)[1: ]
+        # Retrieve the record id to check if the client registered before
+        response_id = retrievePrevRecord(phone_number)
```

Comment:
nit: keep function naming consistent. Change to `retrieve_prev_record` to match the style of other function names